designer/server/src/main/scala/pl/touk/nussknacker/ui/config/UsageStatisticsReportsConfig.scala

```diff
@@ -6,5 +6,7 @@ final case class UsageStatisticsReportsConfig(
     // unique identifier for Designer installation
     fingerprint: Option[String],
     // source from which Nussknacker was downloaded
-    source: Option[String]
+    source: Option[String],
+    // TODO: switch once logstash is ready
+    encryptionEnabled: Boolean = false
```

Comment:
Ended up with `Boolean` here because passing `Option` to `designerConfig` in test causes an error as `Config` is a Java interface